Introduction to Kotlin Multiplatform/04_js.md

```diff
@@ -108,9 +108,12 @@ fun main() {
 Now it is time to start both the server-side JVM process,
```

Comment:
Now it's time to start both the server-side JVM process,
by running the `run` task in Gradle, and the client-side
part by running the `jsRun` task (please note, it may
not work on Windows due to a known bug). Both these Gradle tasks
will run servers for us (a Ktor server for the JVM and a
webpack dev server for the JS), we don't expect these tasks
to complete. Terminating these tasks will stop the dev servers. 


If it all works, we should be able to see something
similar on the web browser